src/pages/CropSidebar/Filters/Filters.js

```diff
@@ -176,7 +182,7 @@ const Filters = ({ filters }) => {
             <Tip filter={filter} />
             <br />
             <DollarsAndRatings
-              state={state}
+              state={filterStateRedux}
```

Comment:
access directly in DollarsAndRatings

Reply:
should we remain the property sent to DollarsAndRatings something like filterState since state doesn't accurately define the property any more.